### PR TITLE
Update readme for the release

### PR DESCRIPTION
**Pull request by yxlao**
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/1848)

### CODE REVIEW

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5539726&bid=18159555)

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5539726&bid=18159555)

File: README.md [hunk 1]
```diff
@@ -4,45 +4,98 @@
...
+**[Homepage](http://www.open3d.org") |**
+**[Docs](http://www.open3d.org/docs") |**
+**[Viewer App](https://github.com/intel-isl/Open3D/releases") |**
+**[Quick Start](http://www.open3d.org/docs/release/getting_started.html") |**
+**[Build from Source](http://www.open3d.org/docs/release/compilation.html") |**
+**[Python API](http://www.open3d.org/docs/release/index.html#python-api-index") |**
+**[C++ API](http://www.open3d.org/docs/release/cpp_api/index.html") |**
+**[Contribute](http://www.open3d.org/docs/release/contribute.html") |**
+**[Demo](https://www.youtube.com/watch?v=I3UjXlA4IsU") |**
+**[Forum](https://forum.open3d.org")**
```
Comment (pvarvak) [inline README.md:16]: all the URLs end with " but do not have " at the start

Comment (yxlao) [reply README.md]: fixed

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5539726&bid=18159555)

**Pull request merged**